rust/src/lib/nm/nm_dbus/connection/vlan.rs

```diff
@@ -15,6 +15,7 @@ pub struct NmSettingVlan {
     pub parent: Option<String>,
     pub id: Option<u32>,
     pub protocol: Option<String>,
+    pub flags: Option<u32>,
```

Comment:
Please use `Vec<NmSettingVlanFlag>` here. You may take `Vec<NmSettingsConnectionFlag>` as example.